service/tests/resources/workspace/Shuttle.toml

```diff
@@ -0,0 +1 @@
+name = "workspace-test"
```

Comment:
Without this a local run of a workspace project will fail, since it will not be able to find a project name anywhere else. I can improve the error message for this as well.

Reply:
This is fixed in #774